prow/oss/plugins.yaml

```diff
@@ -120,6 +120,11 @@ plugins:
   - lifecycle
   - override # Temporarily allow admins to override `pull-release-unit` job that no longer exists.
 
+  google/exposure-notifications-server:
+    - blunderbuss
+    - lifecycle
+    - override # Temporarily allow admins to override `pull-release-unit` job that no longer exists.
```

Comment:
Only keep this if you want repo admins to be able to override job status contexts that are blocking merge.